packages/headless/src/controllers/smart-snippet-questions-list/headless-smart-snippet-questions-list.ts

```diff
@@ -29,10 +29,14 @@ export interface SmartSnippetQuestionsList extends Controller {
   state: SmartSnippetQuestionsListState;
   /**
    * Expand the specified snippet suggestion.
+   *
+   * @param identifier The identifier of a document used to create the smart snippet.
```

Comment:
I think the extractor can process this form, but there would ideally be a dash between the param name and description to adhere to the [tsdoc spec](https://tsdoc.org/pages/tags/param/).